docs_src/security/Ch-Secure-Postgres.md

```diff
@@ -0,0 +1,40 @@
+# Secure PostgreSQL
+
+The superuser password of PostgreSQL will be generated randomly and securely stored in the secret store with security service enabled.
+
+To retrieve the PostgreSQL superuser password, you can use the following methods:
+
+### Obtaining superuser password using the OpenBao CLI
+
+1. Follow the instructions from [Obtaining the OpenBao Root Token](Ch-SecretStore.md#obtaining-the-openbao-root-token) to get the OpenBao root token.
+2. Follow the instructions from [Using the OpenBao CLI](Ch-SecretStore.md#using-the-openbao-cli) to launch the OpenBao CLI.
+3. Retrieve the superuser password by executing the following command in the OpenBao CLI:
+    ```
+    edgex-secret-store:/# bao read secret/edgex/security-bootstrapper-postgres/postgres
+    Key                 Value
+    ---                 -----
+    refresh_interval    168h
+    password            EMletY2JCkOT6lEzZ72f2vo89/hpg/CIcj25Gdk3zMCt
+    username            postgres
+    ```
+
+### Obtaining superuser password using the OpenBao REST API
+
+1. Follow the instructions from [Obtaining the OpenBao Root Token](Ch-SecretStore.md#obtaining-the-openbao-root-token) to get the OpenBao root token.
```

Comment:
Add a description to mention the secret store token from `/tmp/edgex/secrets/security-bootstrapper-postgres/secrets-token.json` can also be used